src/main/java/io/lyuda/jcards/Dice.java

```diff
@@ -0,0 +1,52 @@
+package io.lyuda.jcards;
+
+import java.security.SecureRandom;
+
+/**
+ * The {@code Dice} class represents a dice with an arbitrary amount of sides, defined in the constructor.
+ * This class contains methods to access the number of sides, as well as rolling the dice to generate a random number.
+ *
+ * @author KonnosBaz
+*/
+public class Dice {
+
+    /**
+     * The amount of sides on the dice.
+     */
+    private int sides;
+
```

Comment:
A default (no argument) constructor that creates a standard 6-sided die would be nice.